specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2022-10-01/Tables.json

```diff
@@ -0,0 +1,847 @@
+        }
+      },
+      "description": "Parameters of the search job that initiated this table."
+    },
+    "RestoredLogs": {
+      "type": "object",
+      "properties": {
+        "startRestoreTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The timestamp to start the restore from (UTC)."
+        },
+        "endRestoreTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The timestamp to end the restore by (UTC)."
+        },
+        "sourceTable": {
+          "type": "string",
+          "description": "The table to restore data from."
+        },
+        "azureAsyncOperationId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Search results table async operation id."
+        }
+      },
+      "description": "Restore parameters."
+    },
+    "ResultStatistics": {
+      "type": "object",
+      "properties": {
+        "progress": {
+          "readOnly": true,
+          "type": "number",
+          "multipleOf": 0.01,
+          "description": "Search job completion percentage."
+        },
+        "ingestedRecords": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "The number of rows that were returned by the search job."
+        },
+        "scannedGb": {
+          "readOnly": true,
+          "type": "number",
+          "multipleOf": 0.01,
+          "description": "Search job: Amount of scanned data."
+        }
+      },
+      "description": "Search job execution statistics."
+    },
+    "TableProperties": {
+      "type": "object",
+      "properties": {
+        "retentionInDays": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 4,
+          "maximum": 730,
+          "description": "The table retention in days, between 4 and 730. Setting this property to -1 will default to the workspace retention."
+        },
+        "totalRetentionInDays": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 4,
+          "maximum": 2555,
+          "description": "The table total retention in days, between 4 and 2555. Setting this property to -1 will default to table retention."
+        },
+        "archiveRetentionInDays": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "The table data archive retention in days. Calculated as (totalRetentionInDays-retentionInDays)"
+        },
+        "searchResults": {
+          "$ref": "#/definitions/SearchResults",
+          "description": "Parameters of the search job that initiated this table."
+        },
+        "restoredLogs": {
+          "$ref": "#/definitions/RestoredLogs",
+          "description": "Parameters of the restore operation that initiated this table."
+        },
+        "resultStatistics": {
+          "readOnly": true,
```

Comment:
Updating a field to readonly is a breaking change. Is this already signed off ?

Reply:
Change was made from public preview version and was never stable. breaking change is allowed in this case.